generators/typescript/sdk/generator/src/contexts/type-schema/TypeSchemaContextImpl.ts

```diff
@@ -146,7 +146,7 @@ export class TypeSchemaContextImpl implements TypeSchemaContext {
                       useDynamicImport: false,
                       namespaceImport: "serializers"
                   }
-                : { type: "direct" },
+                : { type: "direct", alias: `${typeName.name.originalName}Type` },
```

Comment:
The alias here is identical to the alias on line 178. If they aren't the same type is imported twice, with different aliases. (Or indeed, if no alias is specified with the conflicting name.)